storm-server/src/main/java/org/apache/storm/localizer/AsyncLocalizer.java

```diff
@@ -109,6 +111,7 @@
         this.blobCacheUpdateDuration = metricsRegistry.registerTimer("supervisor:blob-cache-update-duration");
         this.blobLocalizationDuration = metricsRegistry.registerTimer("supervisor:blob-localization-duration");
         this.numBlobUpdateVersionChanged = metricsRegistry.registerMeter("supervisor:num-blob-update-version-changed");
+        this.localResourceFileNotFound = metricsRegistry.registerMeter("supervisor:local-resource-file-not-found");
```

Comment:
This is very specific to local resource file not found inside `releaseSlotFor`. So the description and the variable should be more specific.

Reply:
fixed